data/json/items/comestibles/seed.json

```diff
@@ -14,6 +14,23 @@
         "symbol": ".",
         "color": "brown"
     },
+    {
+        "id": "seed_bee_balm",
+        "copy-from": "seed",
+        "type": "COMESTIBLE",
+        "name": "bee balm seeds",
+        "name_plural": "bee balm seeds",
+        "description": "Some bee balm seeds.  You could probably plant these.",
+        "charges": 2,
+        "material": "veggy",
```

Comment:
This would be worth looking into now as part of this PR as we could then have seeds as `GENERIC` and not `COMESTIBLE`